lib/grn_windows.h

```diff
@@ -19,6 +19,7 @@
 #pragma once
 
 #include "grn.h"
+#include "wchar.h"
```

Comment:
Why is this included in `windows.c`?

Reply:
Can we move this to `windows.c`?

Reply:
because `wmemmove` requires it.
https://docs.microsoft.com/ja-jp/cpp/c-runtime-library/reference/memmove-wmemmove?view=vs-2019

Reply:
I'll fix it.